docs/reference/advanced_configuration.rst

```diff
@@ -370,7 +370,9 @@ Actions Menu
 ------------
 
 You can add custom items to the actions menu for a specific action by
-overriding the following method::
+overriding the following method
+
+.. code-block:: php
```

Comment:
DOCtor-RST build ask to revert this change

Reply:
:'( ok then